src/resources/business-config.ts

```diff
@@ -14,577 +24,16 @@ export class Business {
   flows: Array<Flow>
 }
 
-// FUTURE: this object needs an interface or type
 export const ConfigJson = [
-  {
```

Comment:
I moved all of this into smaller files and now just import them here for code compatibility.